apps/www/public/rss.xml

```diff
@@ -5,791 +5,799 @@
       <link>https://supabase.com</link>
       <description>Latest news from Supabase</description>
       <language>en</language>
-      <lastBuildDate>Wed, 22 Jun 2022 00:00:00 GMT</lastBuildDate>
+      <lastBuildDate>Mon, 27 Jun 2022 05:00:00 GMT</lastBuildDate>
       <atom:link href="https://supabase.com/blog/rss.xml" rel="self" type="application/rss+xml"/>
       
+<item>
+  <guid>https://supabase.com/blog/2022/06/28/partial-postgresql-data-dumps-with-rls</guid>
+  <title>Partial data dumps using Postgres Row Level Security</title>
+  <link>https://supabase.com/blog/2022/06/28/partial-postgresql-data-dumps-with-rls</link>
+  <description>An example of how to load data into Supabase using supabase-py</description>
+  <pubDate>Mon, 27 Jun 2022 05:00:00 GMT</pubDate>
+</item>
+
 <item>
   <guid>https://supabase.com/blog/2022/06/15/loading-data-supabase-python</guid>
   <title>Python data loading with Supabase</title>
   <link>https://supabase.com/blog/2022/06/15/loading-data-supabase-python</link>
   <description>An example of how to load data into Supabase using supabase-py</description>
-  <pubDate>Fri, 17 Jun 2022 00:00:00 GMT</pubDate>
+  <pubDate>Thu, 16 Jun 2022 05:00:00 GMT</pubDate>
 </item>
 
 <item>
   <guid>https://supabase.com/blog/2022/06/07/beta-update-may-2022</guid>
   <title>Supabase Beta May 2022</title>
   <link>https://supabase.com/blog/2022/06/07/beta-update-may-2022</link>
   <description>Product and community updates including wildcard auth redirects, edge functions with webhooks, and Prometheus endpoints for everybody.</description>
-  <pubDate>Wed, 01 Jun 2022 00:00:00 GMT</pubDate>
+  <pubDate>Tue, 31 May 2022 05:00:00 GMT</pubDate>
```

Comment:
we need to fix this rss feed generator. it keeps using the timezone that the local machine is in